.github/actions/bitcoin-int-tests/Dockerfile.mutation-testing

```diff
@@ -0,0 +1,29 @@
+FROM rust:bullseye
+
+# Set the working directory in the container
+WORKDIR /src
+
+# Copy all the files into the container
+COPY . .
+
+# Update rustup and build the project
+RUN rustup update
```

Comment:
I couldn't combine for example 
`RUN cargo mutants --package clarity --output mutants/clarity` and 
`cargo mutants --package libsigner --output mutants/libsigner` because the first one would return an exit code and stop the whole `RUN`. I am open to suggestions on how to pass that

Reply:
If

```sh
RUN cargo mutants --package clarity --output mutants/clarity \
    cargo mutants --package libsigner --output mutants/libsigner
```

fails I don't see how

```sh
RUN cargo mutants --package clarity --output mutants/clarity
RUN cargo mutants --package libsigner --output mutants/libsigner
```

wouldn't fail also. You can do `|| true` to ignore errors though

Reply:
i would advise not using a dockerfile here, having preference to run the cargo commands locally in a workflow (better if an external composite workflow) step.